### ISSUE TITLE
Upload image to server PathName Android  Network Error

### ISSUE THREAD

**Issue by i3IGm** [RepairShopr/react-native-signature-capture#123]
{type: "image/jpeg", name: "signature", uri: "file:///storage/emulated/0/saved_signature/signature.png"}

**Comment by chevol** [comment]
I am having a similar issue, the image is just nowhere locally. Does your image actually reside in that folder or is it not there by chance?

**Comment by JoseVf** [comment]
Same issue here, but just happens on Android devices, iOS works as expected.

**Comment by chevol** [comment]
Hey JoseVF, I ended up getting it to work but I had to do the following:
Add this Prop to SignatureCapture
`saveImageFileInExtStorage`
Then have some way to specify the file location on different platforms like this:
`const $imgUri = Platform.OS === 'android' ? 'file://' + result.pathName : result.pathName;`

That is important since Android is looking for files  at 'file://' vs just the path on iOS. I hope this helps, let me know if you are still running into issues.

**Comment by JoseVf** [comment]
Sure, as this [comment](https://github.com/RepairShopr/react-native-signature-capture/pull/98#issuecomment-387889261) said...

**Comment by chevol** [comment]
Yes, exactly that. That is where I got it from and it helped me out, sorry was at work and didn't bother to look for where I got the solution from.

**Comment by kirandesai303** [comment]
Did you get any solution?

**Comment by kirandesai303** [comment]
+1
Did you get any solution?

**Comment by Mkay4real** [comment]
> Hey JoseVF, I ended up getting it to work but I had to do the following: Add this Prop to SignatureCapture `saveImageFileInExtStorage` Then have some way to specify the file location on different platforms like this: `const $imgUri = Platform.OS === 'android' ? 'file://' + result.pathName : result.pathName;`
> 
> That is important since Android is looking for files at 'file://' vs just the path on iOS. I hope this helps, let me know if you are still running into issues.

Great!
This works
